md/organization-management-in-bonita-bpm-studio.md

```diff
@@ -74,11 +74,24 @@ Bonita Portal, which could have an impact on deployed processes.
 
 To publish the organization data:
 
-1. Choose **Publish...** from the Organization menu.
-2. Click the organization to be uploaded, and
-then click **_Next_**.
-3. Specify the username and password of the user that will be
+1. Choose **Deploy...** from the Organization menu.
+2. Click the organization to be deployed, and
+3. Specify the username of the user that will be
 used to log in to Bonita Portal when a process is started from
 Bonita Studio. This user must be in the organization you have
 selected.
-4. Click **Publish**.
+4. Click **Deploy**.
+
+## Profile management
+
+In order to log in to bonita UIs (Portal, applications...), a user must have at least on profile (User, Adminstrator...etc).
+
+**_In Community_**, only 3 provided profiles are supported (User, Adminstrator and Process Manager). When deploying an organization from the Studio, each user is automatically mapped to all those profiles.
+
+**_In Teamwork edition_**, those 3 provided profiles can edited in the Studio using an XML editor (Go to Organization > Profiles > Open).
+
+**_In Efficiency and Performance editions_**, profiles can be defined in the Studio using the [Profile Editor](profileCreation.md).
+
+When deploying an organization make sure to synchronize those profiles with the organization. You will be notified when a user is not mapped to any profile while deploying an organization.
```

Comment:
"When creating or importing other organizations, you need to make sure that all users are mapped to at least one profile, through a group, a role, a membership, or as a user, to grant them access to Bonita Portal or applications."

Reply:
IMHO, there is no need to say there are notifications. 
It's super good they are implemented, but they don't need to be promoted in the doc that just needs to say what to do and how to do it.